.github/workflows/build.yml

```diff
@@ -98,10 +98,37 @@ jobs:
         with:
           name: pack-${{ matrix.os }}
           path: out/${{ env.PACK_BIN }}
-  release:
+  build-additional-archs:
     if: ${{ startsWith(github.ref, 'refs/heads/release/') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - goarch: arm64
+          goos: darwin
+          name: macos-arm64
```

Comment:
This may be more than we need now, but I was planning this for expanding eventually to different `arm` architectures